website/versioned_docs/version-0.67/integration-with-android-fragment.md

```diff
@@ -131,12 +242,30 @@ protected void onCreate(Bundle savedInstanceState) {
 }
 ```
 
+</TabItem>
+</Tabs>
+
 In the code above `Fragment reactNativeFragment = new ReactFragment.Builder()` creates the ReactFragment and `getSupportFragmentManager().beginTransaction().add()` adds the Fragment to the Frame Layout.
 
 If you are using a starter kit for React Native, replace the "HelloWorld" string with the one in your `index.js` or `index.android.js` file (it’s the first argument to the AppRegistry.registerComponent() method).
 
 Add the `getLaunchOptions` method which will allow you to pass props through to your component. This is optional and you can remove `setLaunchOptions` if you don't need to pass any props.
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+private fun getLaunchOptions(String message): Bundle {
+    val initialProperties = Bundle()
+    initialProperties.putString("message", message)
+    return initialProperties
+}
```

Comment:
This doesn't compile as it is (`String message` -> `message: String`).

Anyway I would reword to:

```suggestion
private fun getLaunchOptions(message: String) = Bundle().apply {
    putString("message", message)
}
```